pkg/worker/worker_test.go

```diff
@@ -41,9 +41,9 @@ func TestRun(t *testing.T) {
 		})
 	}
 
-	withAggregator(t, expectedResults, func(aggr *aggregation.Aggregator) {
+	withAggregator(t, expectedResults, func(aggr *aggregation.Aggregator, baseURL string) {
 		for _, h := range hosts {
-			url := "http://:" + strconv.Itoa(aggregatorPort) + "/api/v1/results/by-node/" + h + "/systemd_logs.json"
+			url := baseURL + "/api/v1/results/by-node/" + h + "/systemd_logs.json"
```

Comment:
We should move a lot of these strings to consts... 

I still don't like explicit URL composition that isn't hidden under a layer of abstraction.